plaso/dependencies.py

```diff
@@ -32,7 +32,8 @@
     'dfwinreg': ('__version__', '20180712', None, True),
     'dtfabric': ('__version__', '20180707', None, True),
     'efilter': ('', '1.5', None, True),
-    'elasticsearch': ('__versionstr__', '5.4.0', '5.5.1', True),
+    'elasticsearch': ('__versionstr__', '5.4.0', None, False),
```

Comment:
What are your thoughts to make this minimum version 6?

Reply:
I think that's a good idea - updated